.github/workflows/testwindows.yml

```diff
@@ -0,0 +1,62 @@
+name: Test on Windows
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'include'
+      - 'src'
+      - 'tests'
+      - 'tools'
+  workflow_dispatch:
+
+jobs:
+  build-and-asan:
+    name: Test on Windows
+    runs-on: windows-latest
+
+    steps:
+      - name: Intro
+        run: echo "Branch ${{ github.ref }} on repository ${{ github.repository }}"
+
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
3.10 again, not 3.x?

Reply:
Well, you wrote this:

> Luckily, setup-python@v4 is available by now. We also specify a Python version of 3.10 to be on the safe side for recent Meson versions.

and I did not want to 'overrule' that. I guess it does not make much difference. We would also need to update `setup-python` if they release a new version, so there is always something to do with updates.

So in fact, if we prescribe `@4` it could also make sense to prescribe a Python version and not just take the latest one.